arraycontext/fake_numpy.py

```diff
@@ -174,6 +174,32 @@ class BaseFakeNumpyLinalgNamespace:
     def __init__(self, array_context):
         self._array_context = array_context
 
+    def norm(self, ary, ord=None):
```

Comment:
I think there should only be one implementation of `norm`, across here and the pyopencl context.

Reply:
done in https://github.com/inducer/arraycontext/pull/14/commits/609a503f47d2f4bc5e887087992561a5d236ce47